plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -1,12 +1,14 @@
 # Creating and Managing Custom Tables
 
-Welcome to Module 7 of the Intermediate Plugin Developer learning pathway. In this lesson, we will explore how to create, manage, and delete custom database tables in WordPress. As an intermediate developer, this skill is crucial for extending WordPress functionality beyond standard post types and taxonomies.
+As a plugin developer, knowing how to extend WordPress functionality beyond standard post types and taxonomies is a valuable skill to master.
+
+In this lesson, we will explore how to create, manage, and delete custom database tables in WordPress. 
 
 By the end of this lesson, you’ll understand why you might need a custom table, how to create and version the table schema, and how to safely remove a table when it’s no longer needed.
 
 ## Why Create a Custom Table?
 
-In most cases, WordPress's default tables are already enough to store your plugin's data. You can use the posts table to manage different types of content, store user meta to further personalize users' experiences, manage custom settings in the options table, and so much more!
+In most cases, WordPress's default tables are already enough to store your plugin's data. You can use [custom post types](https://learn.wordpress.org/lesson/custom-post-types/) to manage different types of content, store user meta to further personalize users' experiences, manage custom settings in the options table, and so much more!
```

Comment:
Do we not have relevant lessons to hyperlink for "store user meta" and "manage custom settings"?

It seems the previous lesson from the Beginner pathway is relevant, but I know it was mentioned that is a prerequisite to this module. Still, I love when things are hyperlinked so I can quickly refresh my memory or dive deeper.

Possible links I'm seeing...
- User Meta https://github.com/WordPress/Learn/issues/2724 (when it gets published)
- Options https://developer.wordpress.org/apis/options/ (same link from [the beginner pathway](https://learn.wordpress.org/lesson/the-wordpress-database/))

Reply:
You're 100% correct, linking to preexisting lessons or documentation is a great idea. 

What I try and do is only link to lessons that are published, otherwise it means we have to edit this video later on. 

Then I include any links to documentation, or developer blog posts (usually in the conclusion somewhere), because even if those change, the links should remain the same. 

In this case though, I think linking to the developer docs for user meta and options is a good idea. I'll add them.